SimpleHigherHRNet.py

```diff
@@ -90,33 +168,62 @@ def __init__(self,
+        #         print("%d GPU(s) will be used" % torch.cuda.device_count())
+        #         device_ids = None
+        #     else:
+        #         # if device is set to 'cuda:IDS', only that/those device(s) will be used
+        #         print("GPU(s) '%s' will be used" % str(self.device))
+        #         device_ids = [int(x) for x in str(self.device)[5:].split(',')]
+
+        #     self.model = torch.nn.DataParallel(self.model, device_ids=device_ids)
+        # elif 'cpu' == str(self.device):
+        #     print("device: 'cpu'")
+        # else:
+        #     raise ValueError('Wrong device name.')
+        if not trt_:
+            checkpoint = torch.load(checkpoint_path, map_location=self.device)
+            if 'model' in checkpoint:
+                checkpoint = checkpoint['model']
+            # fix issue with official high-resolution weights
+            checkpoint = OrderedDict([(k[2:] if k[:2] == '1.' else k, v) for k, v in checkpoint.items()])
+            self.model.load_state_dict(checkpoint)
+            if 'cuda' in str(self.device):
+                print("device: 'cuda' - ", end="")
+
+                if 'cuda' == str(self.device):
+                    # if device is set to 'cuda', all available GPUs will be used
+                    print("%d GPU(s) will be used" % torch.cuda.device_count())
+                    device_ids = None
+                else:
+                    # if device is set to 'cuda:IDS', only that/those device(s) will be used
+                    print("GPU(s) '%s' will be used" % str(self.device))
+                    device_ids = [int(x) for x in str(self.device)[5:].split(',')]
+                self.model = torch.nn.DataParallel(self.model, device_ids=device_ids)
 
-            if 'cuda' == str(self.device):
-                # if device is set to 'cuda', all available GPUs will be used
-                print("%d GPU(s) will be used" % torch.cuda.device_count())
-                device_ids = None
+            elif 'cpu' == str(self.device):
+                print("device: 'cpu'")
             else:
-                # if device is set to 'cuda:IDS', only that/those device(s) will be used
-                print("GPU(s) '%s' will be used" % str(self.device))
-                device_ids = [int(x) for x in str(self.device)[5:].split(',')]
-
-            self.model = torch.nn.DataParallel(self.model, device_ids=device_ids)
-        elif 'cpu' == str(self.device):
-            print("device: 'cpu'")
+                raise ValueError('Wrong device name.')
+            self.model = self.model.to(device)
+            self.model.eval()
         else:
-            raise ValueError('Wrong device name.')
-
-        self.model = self.model.to(device)
-        self.model.eval()
+            # import pycuda.driver as cuda
+            # self.model = TrtModel('pose_higher_hrnet_w32_512.engine')
+            if device.type == 'cpu':
+                    device = torch.device('cuda:0')
```

Comment:
I think this should be
```suggestion
                raise ValueError('TensorRT does not support cpu device.')
```